addon/serializers/osf-serializer.js

```diff
@@ -99,6 +99,7 @@ export default DS.JSONAPISerializer.extend({
         let documentHash = this._super(...arguments);
         documentHash.meta = documentHash.meta || {};
         documentHash.meta.pagination = Ember.get(payload || {}, 'links.meta');
+        documentHash.meta.total = Math.ceil(documentHash.meta.pagination.total / documentHash.meta.pagination.per_page);
```

Comment:
APIv2 returns total number of *records*, but ember infinity relies on being told the total number of *pages*. 

I can tweak field name and mixin settings to make the name of this field more explicit, if that would help.